metabase/metabase
# handle warnings from reselect about broken selectors rules

Closes https://github.com/metabase/metabase/issues/47087

### Description

Cleanup warnings in console about wrong usage of selectors. Some of the warnings are valid, some are false positives, so we remove noise from them

One warning is valid, and I'll keep it to handle properly later: `const getQuestionsMemoized = createDeepEqualSelector`

### How to verify

- open app in dev mode
- verify console

### Demo

### Before

https://github.com/user-attachments/assets/d714a1d4-b57e-4c83-a3fb-09d5632db6e7

### After


https://github.com/user-attachments/assets/6a7f1efb-7371-4942-b777-11c950be6976
